Sources/Sentry/SentryBreadcrumbTracker.m

```diff
@@ -43,6 +45,20 @@ - (void)dealloc
 }
 #endif // !TARGET_OS_WATCH
 
+// Convenient initializer for tests
+- (instancetype)init
+{
+    return [self initReportAccessibilityIdentifier:YES];
+}
```

Comment:
`m`: This opens a door for initializing the `SentryBreadcrumbTracker` and setting the `_reportAccessibilityIdentifier` to `YES` by mistake. I prefer removing this initializer.